### PR TITLE
Screwed up :(

### PR DESCRIPTION
**Pull request by abaddonpuff**

### CODE REVIEW

File: webCommander/centralHub/views.py [hunk 1]
```diff
@@ -35,12 +36,15 @@ def list_twitter(request):
 def handle_summary(request, twitter_handle):
     user = TwitterUser.objects.get(twitter_handle=twitter_handle)
 
-    #TODO Get posts from database and render them
     posts = user.user_posts.all()
-    breakpoint
     #TODO If no posts get last 20 posts from user
+    # if len(posts) == 0:
+    #Change to 0 after DB gets adjusted
+    if len(posts) == 2:
```
Comment (bbelderbos) [inline webCommander/centralHub/views.py:43]: What is the 2 here?

File: webCommander/centralHub/migrations/0002_twitteruser_twitter_name.py
Comment (bbelderbos) [inline webCommander/centralHub/migrations/0002_twitteruser_twitter_name.py:1]: Why did you end up removing the migration files?

Comment (bbelderbos) [reply webCommander/centralHub/migrations/0002_twitteruser_twitter_name.py:1]: nvm I am reading https://github.com/abaddonpuff/commandCenter/issues/46#issuecomment-2177364308 now

Comment (bbelderbos) [reply webCommander/centralHub/migrations/0002_twitteruser_twitter_name.py:1]: as commented there, let's not merge this so we keep the original migration files, ok?

Comment (abaddonpuff) [reply webCommander/centralHub/migrations/0002_twitteruser_twitter_name.py:1]: Will remove this PR just uploaded for you to check :)

**Pull request closed without merging**